### DISCUSSION TITLE
Mix of box and wide layout together

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#284] [Q&A]
Hello, I have a question about the boxed or wide settings.
I need the Wide setting on a page, because I would like to display the header over the whole page.
The content on the other hand I don't want to display over the whole page, it should be displayed in a kind of box. As an example I have attached screenshot 1 from my old page:
![image](https://user-images.githubusercontent.com/19833064/215566207-aef78091-f5e4-4c11-9f85-b7d39c7ce08c.png)

With Astroid I can do this quite well, by giving the Content Section an image as background, and the column with the Component Area a white color (screenshot 2):
![image](https://user-images.githubusercontent.com/19833064/215566579-df45f61e-3be9-42b7-bd8f-57962385e638.png)

But the problem is that the box should be a little smaller, because my content is not so wide. However, if I display the content in a smaller screen size (smaller than 1200px), then I don't have such thick white borders around my content, then it actually fits.

![image](https://user-images.githubusercontent.com/19833064/215566756-36fb6e21-a58a-4e2c-90d6-a5f4ccf1531c.png)

So I would have to make the Component Area column a bit smaller, but leave the header full width, or make the white borders a bit narrower as in small resolutions.
Maybe you have an idea how I could do that?

WBR,
deltapapa

Translated with www.DeepL.com/Translator (free version)

**Comment by Chacapamac** [comment]
**Hi @deltapapa01 !**
I’m not sure if it will help but you can control settings of the “**Component**” area on multi-devices in the “**Layout**” administration.

![component](https://user-images.githubusercontent.com/1907276/215625842-499ce1d3-3f20-4c5f-b857-8f22f30884c0.jpg)

![Screen Shot 2023-01-30 at 7 16 22 PM](https://user-images.githubusercontent.com/1907276/215626020-6372b15a-c20f-4612-8731-402c1792445f.jpg)

**Comment by deltapapa01** [comment]
Hi,
thank you, but this is not solving my problem.
You can check it here: https://p624550.mittwaldserver.info/cms/
For example, if I change the background only at Komponent, it looks like this:
![image](https://user-images.githubusercontent.com/19833064/216381222-5135a1ed-6b60-47a7-a16b-8b3f2a827fe0.png)
Below 1200 px it looks like this:
![image](https://user-images.githubusercontent.com/19833064/216381473-0915afa2-302d-4144-a50c-15931275974b.png)

But I would like the width of the whole box to look the same above 1200 px as it does below 1200px.

The current live page is here:
https://struckmannonline.de/

WBR,
deltapapa

**Comment by deltapapa01** [comment]
Hello,
I got it solved:
In the custom css file I have entered the following:
`/*grid*/
@media (min-width: 1200px){
	.container, .container-sm, .container-md, .container-lg, .container-xl {
		max-width: 960px !important;
		}
	}`
With this it could be displayed without problems ,as I needed it.

Have a good day,
deltapapa

**Comment by sonvnn** [comment]
You can use theme width option to do this. 
In template Options > Basic > Layout Setttings > Theme Width
![Screen Shot 2023-02-06 at 15 55 43](https://user-images.githubusercontent.com/3204103/216928160-39b339bc-5c50-4314-a2d1-e5ea45e1f8ba.png)

Best Regards,
Sonny

**Comment by deltapapa01** [comment]
Hi Sonny,
thanks for the info, but that doesn't work in my case, because then the logo section is also "boxed", and I want to have this area over the whole screen.
Under struckmannonline.de I have the new page online now, and it looks really good with J4 and Astroid Framework.

Thanks a lot,
deltapapa

**Comment by Chacapamac** [comment]
I see now, thank You @deltapapa01 , I will put that in my Astroid code library, in case I need it.
Good Day!
